src/pmp/visualization/SurfaceMeshGL.cpp

```diff
@@ -40,7 +41,7 @@ SurfaceMeshGL::SurfaceMeshGL()
 
     // material parameters
     front_color_ = vec3(0.6, 0.6, 0.6);
-    back_color_ = vec3(0.5, 0.0, 0.0);
```

Comment:
Why this change? The red rendering of back faces is quite useful, e.g., for detecting intersections or fold-overs.

Reply:
don't know anymore. something didn't work. let's change it back.

Reply:
Okay, I just pushed a change for that.